concepticondata/references/references.bib

```diff
@@ -3163,3 +3163,13 @@ @article{Lee2015
   journal      = {{PLOS} {ONE}}
 }
 
+@incollection{Urban2016,
+    author     = {Urban, Matthias},
+    year       = {2016},
+    title      = {Motivation by formally analyzable terms in a typological perspective: An assessment of the variation and steps towards explanation},
+    booktitle  = {The lexical typology of semantic shifts},
+    pages      = {555–576},
+    publisher  = {De Gruyter},
```

Comment:
I think they tend to spell it now "DeGruyter". And you should add teh address as well. This would be `address = {Berlin and New York}`.